app/forms/questionnaire_form.py

```diff
@@ -104,10 +104,10 @@ def _validate_calculated_question(self, question, target_answer, target_total):
     def _get_period_limits(limits):
         minimum, maximum = None, None
         if limits:
-            if 'min_value' in limits:
-                minimum = limits['min_value']
-            if 'max_value' in limits:
-                maximum = limits['max_value']
+            if 'minimum_days_limit' in limits:
+                minimum = limits['minimum_days_limit']
+            if 'maximum_days_limit' in limits:
```

Comment:
This hasn't addressed my previous comment. "We shouldn't limit ourselves to just days. Years, Months, Weeks and Days could all be valid requirements in the future." This can quite easily be done by having a structure as below and using **dateutil.relativedelta**.
```
"min": {
   "days": 23
}

```
or 

```
"min": {
   "months": 3
   "years": 1
}
```

Reply:
@dwyeradam I did reply on your last change. There is no requirements for month and year in between a date range. I have double checked with Donna also everything upcoming is for days in between, the trello card also describes days as the only requirement. Also months/years can be representing in days also. 
In regard the `relativedelta` I am just in process of changing over but its causing some issues so still working througth that

Reply:
@shahi645 it may not be required now, but if it is in the future then we are going to end up having a messy schema.
This pattern has already been introduced so it seems sensible to me to keep this kind of definition consistent across the schema.

Reply:
@shahi645. I agree with Jon. We're building a feature not a survey. That feature would be better written allowing for more than just days. Also the code required for that does not add any additional complication. As I keep saying if you research the use of relativedelta you'd realise days, months, years, weeks can all be done in a single line.

Reply:
Yes i don't disagree with that i have looked in to relativedelta its simple enough im working on those changes currently. In regards to the days/weeks/months there is no future requirement for it. This is the exact same thing that was said for average on the calculated question type and it just got backed out in the end.